test/SpokePoolClient.ValidateFill.ts

```diff
@@ -122,6 +122,16 @@ describe("SpokePoolClient: Fill Validation", function () {
     expect(filled.eq(deposit.amount)).is.true;
   });
 
+  it.skip("Tracks v3 fill status", async function () {
```

Comment:
did you mean to leave this as skip-ed?

Reply:
Yeah - we need to bump contracts before we can test this because it queries the SpokePool contract directly (bypassing the SpokePoolClient).